src/webportal/src/app/job/job-view/fabric/task-attempt/task-attempt-list.jsx

```diff
@@ -0,0 +1,644 @@
+                  result.push('');
+                  // content
+                  result.push('[Exit Diagnostics]');
+                  result.push('');
+                  result.push(diag);
+                  result.push('');
+                }
+
+                this.setState({
+                  monacoProps: {
+                    language: 'text',
+                    value: result.join('\n'),
+                    options: {
+                      wordWrap: 'off',
+                      readOnly: true,
+                    },
+                  },
+                  monacoTitle: `Task Exit Diagonostics`,
+                });
+              }}
+            />
+          );
+        },
+      },
+      {
+        key: 'containerId',
+        name: 'Container ID',
+        headerClassName: FontClassNames.medium,
+        minWidth: 300,
+        isResizable: true,
+        onRender: item => {
+          const id = item.containerId;
+          return (
+            !isNil(id) && (
+              <div className={c(t.truncate, FontClassNames.mediumPlus)}>
+                {id}
+              </div>
+            )
+          );
+        },
+      },
+      {
+        key: 'info',
+        name: 'Info & Logs',
+        className: localCss.pa0I,
+        headerClassName: FontClassNames.medium,
+        minWidth: 300,
+        maxWidth: 500,
+        onRender: item => (
+          <div
+            className={c(t.h100, t.flex, t.justifyStart, t.itemsCenter, t.ml1)}
+          >
+            <div className={c(t.flex)} style={{ height: 40 }}>
+              <CommandBarButton
+                className={FontClassNames.mediumPlus}
+                styles={{
+                  root: { backgroundColor: 'transparent' },
+                  rootDisabled: { backgroundColor: 'transparent' },
+                }}
+                iconProps={{ iconName: 'TextDocument' }}
+                text='Stdout'
+                onClick={() =>
+                  this.showContainerLog(
+                    `${item.containerLog}user.pai.stdout`,
+                    'stdout',
+                  )
+                }
+                disabled={isNil(item.containerId) || isNil(item.containerIp)}
+              />
+              <CommandBarButton
+                className={FontClassNames.mediumPlus}
+                styles={{
+                  root: { backgroundColor: 'transparent' },
+                  rootDisabled: { backgroundColor: 'transparent' },
+                }}
+                iconProps={{ iconName: 'Error' }}
+                text='Stderr'
+                onClick={() =>
+                  this.showContainerLog(
+                    `${item.containerLog}user.pai.stderr`,
+                    'stderr',
+                  )
+                }
+                disabled={isNil(item.containerId) || isNil(item.containerIp)}
+              />
+            </div>
+          </div>
+        ),
+      },
+    ];
+    const columns = defaultColumns;
+    return columns;
```

Comment:
>columns [](start = 11, length = 7)

order the cols the same as job detail page?

Reply:
I think ```IP and Ports``` is not important in job attempt list, so I put them behind. Or could we just remove them?

Reply:
done